src/main/java/org/jabref/gui/exporter/ExportToClipboardAction.java

```diff
@@ -115,12 +120,20 @@ private String exportToClipboard(Exporter exporter) {
                 }
             }
         }
-        return "";
+        throw new Exception("export failed.");
```

Comment:
I think then we should also remove the `Exception` handler above (or rethrow there) to get a more detailed exception message of what went wrong.

Reply:
As the Exception is already logged above, it's not necessary to get a more detailed message - however, I added the `onFailure()` to swallow the rethrow in 2036ec2